### DISCUSSION TITLE
Metadata not displayed in icecast player

### DISCUSSION THREAD

**Discussion by fsan2val** [AzuraCast/AzuraCast#6524] [General]
I have tried various player plugins (free and paid player ) for icecast on website wordpress but none read metadata.

Even in its azuracast demo I have taken the transmission link but it still does not show the song title or cover

Icecast does not work, any solution

**Comment by Piratenkanon** [comment]
HTML5-Player for Icecast-Stream

1. modify in index.html line 7 your website-title
2. modify in js/player.xml line 12 your stream-url
3. modify in js/player.xml line 17 the Radioname shown in player
4. modify in js/currentsong.php line 49 your stream-url

you can change the style of the player in the css index.html line 19-32

in wordpress you can add the player with HTML-code:
<iframe width="90%" height="100" scrolling="no" frameborder="0" src="https://your-domain.com/player-icecast"></iframe>


you can download the player here:
https://workupload.com/file/mFA2Kgpmun8  or
https://usersdrive.com/zszb31p7dkk1.html
